content/en/docs/setup/install/virtual-machine/index.md

```diff
@@ -31,52 +31,24 @@ but not production. Like all alpha features, this guide is subject to change.
 1. Virtual machines must have IP connectivity to the ingress gateway in the connecting mesh, and optionally every pod in the mesh via L3 networking if enhanced performance is desired.
 
 ## Prepare the guide environment
-
-1. Set the environment variables `"${ISTIO_DIR}"`, `"${WORK_DIR}"`, your cluster
-    name, and the service namespace. Ensure `"${WORK_DIR}"` is prefixed with `"${HOME}"`
+1. Create a VM in one of the cloud provider
+1. Set the environment variables `"${VM_NAME}"`, `"${WORK_DIR}"`,`"${CLUSTER_NAME}"` , `"${SERVICE_ACCOUNT}"`,
```

Comment:
what does VM_NAME represent? a short explanation would be helpful.

Reply:
same story for SERVICE_ACCOUNT

I know this would be improving upon the existing documentation, but a table format of each variable and what it means would be preferable to a long list of variables without much context.

Reply:
the virtual machine means the name when you create the vm instance. It is not required. It is just  an identifier or alias when you demo